### DISCUSSION TITLE
0.6.0 release

### DISCUSSION THREAD

**Discussion by vincentsarago** [developmentseed/geojson-pydantic#112] [General]
initial we planned to do a 0.6 release with all the latest changes 

https://github.com/developmentseed/geojson-pydantic/blob/96eb47265e55d7d429ef274d9eb11e0aacf0b546/CHANGELOG.md?plain=1#L8-L50

I think those changes are so important that we should do a 1.0 instead

any though? @eseglem @geospatial-jeff

**Comment by eseglem** [comment]
Might be worth putting everything out as a 0.6 and let users provide some feedback given the number of breaking changes. Maybe a 0.6 and then an rc1 at some set time in the future? 

We could also consider waiting on a Pydantic v2 release to see if we can fix any of the remaining issues / PRs. I don't know how others are going to handle it, but I don't know if having a change like that on a minor release makes sense. Feels more like a major release to me. 

I do have one more idea for `BBox` and being able to add a little more enforcement to it. And if it works out, would probably be good to get it in the same release. Whatever it is. So I'll try and throw that together soon.

**Reply by vincentsarago** [reply]
> We could also consider waiting on a Pydantic v2 release 

oh forget about this 🤦, yeah let's wait until then. 

> Just an update that we're aiming for an alpha release of Pydantic V2 by the end of March.

from https://github.com/pydantic/pydantic/issues/4887#issuecomment-1468022094

or we could do 0.6 with what we have right now, and do a 1.0 when we switch to pydantic 2.0 

in any case we should do an alpha release first 👍

**Comment by vincentsarago** [comment]
made a `0.6.0a0` release https://pypi.org/project/geojson-pydantic/0.6.0a0/

**Comment by vincentsarago** [comment]
FYI I just pushed `0.6` https://github.com/developmentseed/geojson-pydantic/releases/tag/0.6.0
